File: 344-reverse-string/reverse-string.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """

        l = 0
        h = len(s) - 1

        while l < h:
            temp = s[l]
            s[l] = s[h]
            s[h] = temp

            l += 1
            h -= 1

        
        

File: 344-reverse-string/reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        char temp;

        int l = 0;
        int h = s.length - 1;

        while (l < h) {

            temp= s[l];
            s[l] = s[h];
            s[h] = temp;

            l++;
            h--;
        }
    }
}